src/domains/transaction/components/TransactionDetailModal/TransactionDetailModal.tsx

```diff
@@ -72,7 +72,7 @@ export const TransactionDetailContent = ({
 						network={transaction.wallet().network()}
 						recipients={recipients.map(({ address, alias }) => ({
 							address,
-							alias,
+							alias: isContractTransaction(transaction) ? undefined : alias,
```

Comment:
this just sets any contract to have no alias, so if you were to add a contract as a contact in the wallet it wouldn't show up either 🤔 why was it showing the wrong alias in the first place? shouldn't the alias be based on the recipient and therefore show the right value irregardless of it being a contract or regular address?